### ISSUE TITLE
Reproducibility issue: performance lower than reported in paper

### ISSUE THREAD

**Issue by ddww-dong** [Sutuk/FracFormer#3]
Dear authors,

Thank you for your excellent work and for releasing the code.

I am trying to reproduce the results reported in the paper, but my performance is significantly lower than the numbers reported in Table Ⅱ.

Environment:
- GPU: 2080ti x 4
- CUDA: 11.3
- PyTorch: 1.7.0+cu101
- Batch size: 40
- Training epochs: 500
- Learning rate: 1e-4
- Weight_decay=5e-4
- Dataset: I applied the data augmentation strategy described in the paper to the dataset(Femur) provided in the GitHub repository, generating 100 augmented samples for each original instance. The dataset was then split into training and validation sets using a 7:3 ratio。

Below are the training curves I obtained.
<img width="952" height="714" alt="Image" src="https://github.com/user-attachments/assets/c4007ec7-f877-4024-bbe0-49dd0604299a" />

<img width="952" height="714" alt="Image" src="https://github.com/user-attachments/assets/06546edf-bfd6-439a-ad28-5cfa81bfc384" />
I observed that the loss became NaN at epoch 451 during training.

<img width="457" height="368" alt="Image" src="https://github.com/user-attachments/assets/225ce652-b877-4c56-aa87-4532b1132119" />

Additionally, below are some results from my validation set.

=== Average over 200 samples ===
Average Translation Error (mean): 7.284833(mm)

Average Rotation Error (mean deg): 28.1547°

Average L2 Chamfer Distance (Restored Scale): 27.867869(mm)

Threshold 0.10 → Average PA: 1.0000
Threshold 0.05 → Average PA: 0.9988
Threshold 0.02 → Average PA: 0.9742

The final visualization results indicate that the global position is approximately recovered. However, the fractured interfaces are not precisely aligned, and the rotation error is still relatively high, even after applying the ICP refinement strategy described in the paper.

I might have misunderstood some implementation details. Could you please advise on how to achieve more accurate results? Additionally, if available, would it be possible to share a pretrained model for reference?

I would sincerely appreciate your assistance.
